src/app/workflow/workflow.component.html

```diff
@@ -244,12 +244,17 @@ <h3>
             ></app-info-tab>
           </mat-tab>
           <div *ngIf="entryType === EntryType.Notebook">
-            <mat-tab id="codeTab" label="Code">
-              <div class="mt-4 ml-4 mr-4">
-                <p>A user-friendly representation of the notebook will appear here, after it is implemented in SEAB-5117.</p>
-                <p>For a Jupyter notebook, it'll be the contents of the code/documentation cells, nicely-formatted.</p>
-                <p>For other kinds of notebooks, it might be something different.</p>
-              </div>
+            <mat-tab id="previewTab" label="Preview">
+              <ng-template matTabContent>
+                <div class="mt-4 ml-4 mr-4">
+                  <app-notebook
+                    *ngIf="workflow && selectedVersion && (extendedWorkflow$ | async)"
+                    [notebook]="workflow"
```

Comment:
1. I thought we had an issue for this, but we don't, so I created dockstore/dockstore#5471.
2.  Looking at the IDE errors just caught a legit problem [here](https://github.com/dockstore/dockstore-ui2/pull/1756#discussion_r1170582877).
3. I entered the original comment, so I'm biased, but I don't think we should introduce additional errors, even though in the case, it's the same underlying JS object, it won't be a runtime failure. It just make things cleaner for the future.

Another optional way to fix it, instead of changing the import, is to allow both Swagger and OpenAPI for the input in notebook.component.ts.